ui/src/components/CreateOrEditTask.vue

```diff
@@ -186,6 +186,10 @@
             >{{ $t("tasks.every") }} 5
             {{ $t("tasks.minutes") }}</cv-dropdown-item
           >
+          <cv-dropdown-item value="10m"
+            >{{ $t("tasks.every") }} 10
```

Comment:
Other languages might translate this with a different word order and syntax. Let's use a single i18n string with a parameter for every dropdown item, e.g.:
```js
$t("tasks.every_minutes", { num: 10 })
```
In `translation.json`:
```json
"tasks": {
  "every_minutes": "Every {num} minutes"
}
```